### ISSUE TITLE
[Audio] Byahkee does not play any "roar" sound during climax.

### ISSUE THREAD

**Issue by YoWaddupFD** [itsamewayu/OLFallenDoll#816]
All xenos have a climax noise that is played within an experiment, and the Byahkee's seems to be missing. in both 3.4 and 4.9, the roar was present. It is missing in 7.4.

**Comment by yuanyuankuang** [comment]
No,the  noise is  ejaculating outside. IN 23s,the sound is  ejaculating outside.
